### PR TITLE
fix: changing.secret account key.in.secret

### PR DESCRIPTION
**Pull request by aleksandrmelnikov**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes onepanelio/core#

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/config_types.go [hunk 1]
```diff
@@ -275,7 +276,7 @@ func (g *ArtifactRepositoryGCSProvider) MarshalToYaml() (string, error) {
 			Endpoint:  g.Endpoint,
 			Insecure:  g.Insecure,
 			ServiceAccountKeySecret: ArtifactRepositorySecret{
-				Key:  "serviceAccountKey",
+				Key:  "artifactRepositoryGCSServiceAccountKey", //artifactRepositoryGCSServiceAccountKey //serviceAccountKey
```
Comment (rushtehrani) [inline pkg/config_types.go:279]: Are the comments necessary?

File: pkg/workflow_template.go [hunk 2]
```diff
@@ -295,6 +295,21 @@ func (c *Client) getWorkflowTemplate(namespace, uid string, version int64) (work
 
 	workflowTemplate.Labels = labelsMap[workflowTemplate.WorkflowTemplateVersionID]
 
+	params, err := workflowTemplate.GetParametersKeyString()
```
Comment (rushtehrani) [inline pkg/workflow_template.go:298]: Is this supposed to be here or in a different PR?

File: pkg/config_types.go (hunk 1 above)
Comment (aleksandrmelnikov) [reply pkg/config_types.go]: No, they are not. Removing.

File: pkg/workflow_template.go (hunk 2 above)
Comment (aleksandrmelnikov) [reply pkg/workflow_template.go:298]: Different PR, I branched off the wrong branch. Closing this PR, use this one instead. https://github.com/onepanelio/core/pull/425
